code/modules/mod/modules/modules_general.dm

```diff
@@ -514,8 +514,8 @@
 			/obj/item/clothing/head/nursehat,
 			/obj/item/clothing/head/chefhat,
 			/obj/item/clothing/head/papersack,
-			)) - /obj/item/clothing/head/caphat/beret
-			//Need to subtract the beret because its annoying
+			/obj/item/clothing/head/caphat/beret
```

Comment:
you need to make skyrat edit comments for this

Reply:
mhm, I will go make a pr there for it